inspirehep/modules/authors/utils.py

```diff
@@ -66,6 +68,9 @@ def bai(name):
         last_name = names[0]
         initials = []
 
+    # FIXME: This should be loaded on the outer level when travis is updated.
+    from beard.utils.strings import asciify
```

Comment:
@Panos512 can you cut & paste from `beard` the `asciify` function into `utils.py` as in:

```python
try:
    from beard.utils.strings import asciify
except ImportError:
    def asciify(...):
         ....
```

In this way we don't depend necessarily on `beard` but if it's there we use it :-)